central/declarativeconfig/manager_impl_test.go

```diff
@@ -221,71 +269,6 @@ func TestReconcileTransformedMessages_ErrorPropagatedToReporter(t *testing.T) {
 	}
 }
 
-func TestReconcileTransformedMessages_MissingUpdaterCausesPanic_DevBuild(t *testing.T) {
```

Comment:
Should we still test logic around `verifyUpdaters`? I think we can do that by using the fact that the reporter will not be called if it returns an error.

Reply:
Added tests for `verifyUpdaters` specifically.